anyblok_pyramid/bloks/pyramid/restrict.py

```diff
@@ -0,0 +1,72 @@
+# This file is a part of the AnyBlok / Pyramid project
+#
+#    Copyright (C) 2002 Pierre Verkest <pierreverkest84@gmail.com>
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file,You can
+# obtain one at http://mozilla.org/MPL/2.0/.
+from anyblok.common import add_autodocs
+from anyblok.model.plugins import ModelPluginBase
+
+
+class RestrictQueryByUserIdPlugin(ModelPluginBase):
+    """An AnyBlok plugin that helps to add extra filters on queries according
+    the current user"""
+
+    def __init__(self, registry):
+        if not hasattr(registry, "restrict_query_by_user_methods"):
+            registry.restrict_query_by_user_methods = {}
+
+        super().__init__(registry)
+
+    def transform_base_attribute(
```

Comment:
I hate this presentation of the code

Reply:
this looks like black signature :) I think they choose this design to let reviewer focus on real changes, but as long we are not using black here I'll change that !